### PR TITLE
LP-2522 multilingual course groups

### PR DESCRIPTION
**Pull request by asadali145**
[LP-2522](https://philanthropyu.atlassian.net/browse/LP-2522)

### CODE REVIEW

**Comment by zahid-arbisoft** [comment]
## Diff Coverage Report - Edx
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/1145/workflows/workflows_id/jobs/1145/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/master...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/applications/admin.py (87.5%): Missing lines 144,170-171,173
openedx/adg/lms/applications/constants.py (100%)
openedx/adg/lms/applications/helpers.py (94.1%): Missing lines 301
openedx/adg/lms/applications/management/commands/tests/test_update_is_prerequisite_courses_passed.py (100%)
openedx/adg/lms/applications/management/commands/update_is_prerequisite_courses_passed.py (100%)
openedx/adg/lms/applications/managers.py (100%)
openedx/adg/lms/applications/migrations/0006_auto_20210222_0532.py (0.0%): Missing lines 3-4,7,9,14
openedx/adg/lms/applications/models.py (100%)
openedx/adg/lms/applications/tests/factories.py (100%)
openedx/adg/lms/applications/tests/test_admin/test_inlines.py (100%)
openedx/adg/lms/applications/tests/test_helpers.py (100%)
openedx/adg/lms/applications/tests/test_models.py (100%)
openedx/adg/lms/applications/views.py (100%)
-------------
Total:   199 lines
Missing: 10 lines
Coverage: 94%
-------------
```

File: openedx/adg/lms/applications/admin.py [hunk 1]
```diff
@@ -122,14 +127,59 @@ class WorkExperienceAdmin(admin.ModelAdmin):
...
 
-@admin.register(PrerequisiteCourseGroup)
-class PrerequisiteCourseGroupAdmin(admin.ModelAdmin):
-    inlines = [PrerequisiteCourseAdmin, ]
-    list_display = ('name', 'prereq_course_count', 'open_prereq_courses_count',)
+
+@admin.register(MultilingualCourseGroup)
+class MultilingualCourseGroupAdmin(admin.ModelAdmin):
+    inlines = (MultilingualCourseAdmin,)
+    list_display = ('name', 'is_prerequisite', 'multilingual_course_count', 'open_multilingual_courses_count')
```
Comment (MunamMubashir-arbisoft) [inline openedx/adg/lms/applications/admin.py:182]: Kindly make use of trailing comma consistent; either remove it from everywhere or add it everywhere.

**Review by MunamMubashir-arbisoft — APPROVED** [review]
Great work with refactoring the tests!
Added one minor comment, kindly take a look. All else looks good.

File: openedx/adg/lms/applications/admin.py (hunk 1 above)
Comment (asadali145) [reply openedx/adg/lms/applications/admin.py:182]: Isn't it consistent? At line 181, the comma is required to define a tuple as it has a single element but for line 182 we don't need comma. For tuples with single element comma is required to differentiate them but for multiple elements it isn't required.

Comment (asadali145) [reply openedx/adg/lms/applications/admin.py:182]: Also the trailing comma is helpful if a tuple, list or a dict spans over multiple lines.

Comment (MunamMubashir-arbisoft) [reply openedx/adg/lms/applications/admin.py:182]: We've been using a trailing comma in some of our code. I thought maybe we could make a convention either way.

Comment (asadali145) [reply openedx/adg/lms/applications/admin.py:182]: You are right, We can have some convention on this as well.

**Pull request merged**